apis/Google.Devtools.AspNet/Google.Devtools.AspNet.IntegrationTests/CloudErrorReporting/CloudErrorReportingExceptionLoggerIntregrationTest.cs

```diff
@@ -29,48 +28,42 @@ namespace Google.Devtools.AspNet.IntegrationTests
 {
     public class CloudErrorReportingExceptionLoggerIntregrationTest
```

Comment:
Given that these are all integration tests and we're in the CloudErrorReporting namespace, I'd just call this ExceptionLoggerTest